composer.json

```diff
@@ -38,6 +38,7 @@
         "ext-pcre": "*",
         "ext-reflection": "*",
         "ext-simplexml": "*",
+        "jean85/pretty-package-versions": "^1.6 || ^2.0",
```

Comment:
IIRC, `v1.6` is needed for `composer:v1` only.
Given the [steady decline](https://packagist.org/packages/jean85/pretty-package-versions/stats) of `v1` in favor of `v2`, I'd say let's push `v2` only :rocket: